hs/Response/Draw.hs

```diff
@@ -6,12 +6,15 @@ module Response.Draw
 import           Text.Blaze ((!))
 import qualified Text.Blaze.Html5 as H
 import qualified Text.Blaze.Html5.Attributes as A
+import qualified Data.Text as T
+
 import Happstack.Server
 import MasterTemplate
 import Scripts
 
+
 drawResponse :: ServerPart Response
-drawResponse =
+drawResponse = do
```

Comment:
@R0ss the `do` is not necessary because there isn't any monadic chaining here. @Ian-Stewart-Binks went through and deleted all the unnecessary `do` keywords, so let's keep them removed.